pom.xml

```diff
@@ -59,6 +60,12 @@
     <maven.dependency.plugin.version>3.1.1</maven.dependency.plugin.version>
     <nexus.staging.maven.plugin.version>1.6.8</nexus.staging.maven.plugin.version>
     <lifecycle.mapping.version>1.0.0</lifecycle.mapping.version>
+
+    <!-- EG specific plugin properties -->
+    <spotbugs.config>file://${project.build.directory}/plugin-config/spotbugs/eg-spotbugs-exclude.xml</spotbugs.config>
```

Comment:
See my comments on the similar hotels-oss PR. Let's try get it all working there and then can update this to be the same.